core/core_modules/admin.js

```diff
@@ -0,0 +1,167 @@
+﻿var config = require.once('../config.js'),
+
+getHasPermission = function (config, userId, name, threadId, moduleName) {
+    if (!config.modules || !config.modules[moduleName] || config.modules[moduleName].length === 0) {
+        return true;
+    }
+
+    if (!config.users || Object.keys(config.users).length === 0) {
+        return false;
+    }
+
+    var user = config.users[userId] || config.users[name];
+    if (!user || !user[threadId] || user[threadId].length === 0) {
+        return false;
+    }
+
+    var common = user[threadId].filter(function (p) {
+        return config.modules[moduleName].indexOf(p) !== -1;
+    });
+
+    return common.length >= 1;
+},
+
+modify = function (name, threadId, action, method) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 space characters but found 0.](https://www.codacy.com/app/theprojectshop/Kassy/file/2560099488/issues/source?bid=3282987&fileBranchId=3292072#l24)